security/libfido2/Portfile

```diff
@@ -3,8 +3,9 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           openssl 1.0
```

Comment:
if you add the `openssl` PG then you should remove the `path:lib/libssl.dylib:openssl` dependency

Reply:
Ok thanks!

Reply:
also, please squash your commits into one and make sure the commit message follows our guidelines (i.e., `portname: short description`.

Reply:
I think I got it now